File: silver/11866/c++/main.cpp
#include <iostream>
#include <vector>
#include <queue>


int main()
{
    int n, k;
    std::cin >> n >> k;

    std::queue<int> _queue;
    std::vector<int> array;

    int index = 0;
    int data;
    
    for (int i = 1; i <= n; i++) {
        _queue.push(i);
    }

    while (! _queue.empty()) {
        for (int i = 0; i < k -1; i++) {
            _queue.push(_queue.front());
            _queue.pop();
        }
        array.push_back(_queue.front());
        _queue.pop();
    }

    std::cout << '<';
    for (int i = 0; i < array.size(); i++) {
        if (i > 0) {
            std::cout << ", ";
        }
        std::cout << array[i];
    }
    std::cout << '>';


    return 0;
}

File: silver/11866/python/main.py

if (__name__ == "__main__"):
    n, k = map(int, input().split())

    array = list()
    queue = list()

    for i in range(1, n +1):
        queue.append(i)

    while (len(queue)):
        for i in range(k -1):
            data = queue.pop(0)
            queue.append(data)
        array.append(queue.pop(0))
    print('<', end='')
    print(", ".join(map(str, array)), end='')
    print('>')